helix-term/src/commands.rs

```diff
@@ -1143,6 +1165,13 @@ mod cmd {
             fun: format,
             completer: None,
         },
+        Command {
+            name: "indent_style",
```

Comment:
`indent-style`, commands use dashes

Reply:
Kakoune uses `indent-width 0-8` where `0` means "use tabs". I think we can keep the current behavior because it's clearer

Reply:
Can you also add a `tab-width` for controlling `doc.tab_width()`? This is simply for rendering purposes

Reply:
> Kakoune uses `indent-width 0-8` where `0` means "use tabs".

It's super easy to make `0` an alias for tab, just for people's muscle memory.  I'll add that.

> Can you also add a `tab-width` for controlling `doc.tab_width()`?

Happy to do that, but does that property belong on the document?  I feel like tab width is more of an editor-wide preference, since it's purely a display thing.

What do you think?  I can take care of that as a separate PR, moving where the setting lives and adding a command to change it.

Reply:
I think that's fine 👍🏻